app/coffee/RedisManager.coffee

```diff
@@ -24,16 +31,21 @@ module.exports = RedisManager =
 		logger.log project_id:project_id, doc_id:doc_id, version: version, hash:docHash, "putting doc in redis"
 		ranges = RedisManager._serializeRanges(ranges)
 		multi = rclient.multi()
-		multi.set keys.docLines(doc_id:doc_id), docLines
+		multi.eval setScript, 1, keys.docLines(doc_id:doc_id), docLines
```

Comment:
Does the redis driver handle sending the script once, and then just calling it by SHA? It's not a big script, so this probably doesn't matter either way.